config.default.yaml

```diff
@@ -162,6 +163,17 @@ solar_thermal:
     slope: 45.
     azimuth: 180.
 
+existing_capacities:
```

Comment:
Can you elaborate what the effect of "grouping_years_power" is and what values are necessary/allowed?

Reply:
Assets are installed with "continuous" years, e.g. 1960, 1961, 1962 etc.
The info are parsed and grouped at custom "bins" years like histogram and this option grouping years specify the "bins".
This is then used to clarify the lifetime of the components in a more "discrete" approach according to the modeler will to define the planning_horizons.
Not sure if that's clear

Reply:
It is more or less clear yes, how should we handle this information? Small comment in the config, since the docs are not activate for earth-sec